udf/udf_kv_array_to_json_string.sql

```diff
@@ -0,0 +1,43 @@
+/*
+
+Returns a JSON string representing the inputed key-value array.
```

Comment:
```suggestion
Returns a JSON string representing the inputted key-value array.
```

Reply:
Plain old "input key-value array" would seem most natural here to me.